src/main/java/com/elixr/poc/rest/controller/UserDeletionController.java

```diff
@@ -24,23 +24,16 @@ public UserDeletionController(UserService userService) {
     }
 
     /**
-    Calling deleteUserDetails method with the parameter userId to delete the user by userId.
-    And handling the Exception if the userId is not matching.
+     * Calling deleteUserDetails method with the parameter userId to delete the user by userId.
+     * And handling the Exception if the userId is not matching.
      */
     @DeleteMapping("/user/{userId}")
-    public ResponseEntity<?> deleteUser(@PathVariable("userId") UUID userId) {
-        DeleteResponse deleteResponse = new DeleteResponse();
+    public ResponseEntity deleteUser(@PathVariable("userId") @Valid UUID userId) {
+        CommonErrorResponse commonErrorResponse;
```

Comment:
Better to create CommonResponse and  success case not need to add errormessage jues message

Reply:
Updated the 1st part of the comment, but as per requirement we need to show message as "errorMessage"